Firmware/Marlin_main.cpp

```diff
@@ -1,9645 +1,9649 @@
-/* -*- c++ -*- */
```

Comment:
It looks like you have changed 19294 lines in Marlin_main.cpp this is probably not intended, check line endings CR+LF vs LF